### ISSUE TITLE
Feature request: check single line assignments

### ISSUE THREAD

**Issue by rpbaptist** [rubocop-hq/rubocop#5926]
## Expected behavior

I am trying to get the following situation checked by Rubocop:

```ruby
# bad
foo =
  :bar

# good
foo = :bar
```

I was hoping configuring `EnforcedStyle: same_line` in `Layout/MultilineAssignmentLayout` would catch this, but it only looks at multiline.

I would be willing to contribute, probably on a new more simple cop. Something like `Layout/SingleLineAssignment`? Would this be useful for Rubocop?

**Comment by rpbaptist** [comment]
This is my own duplicate :man_facepalming: 

https://github.com/rubocop-hq/rubocop/issues/5673
